src/components/Spinner/Spinner.st.css

```diff
@@ -0,0 +1,60 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:vars {
+    DefaultPathColor: color(color-5);
+}
+
+:vars {
+    PathColor: --overridable;
+}
+
+:vars {
+    UsedPathColor: color(fallback(value(PathColor), value(DefaultPathColor)));
+}
+
+.root {
+    -st-states: mobile, centered, static;
+
+    animation: rotate 2s linear infinite;
+    z-index: 2;
```

Comment:
Not sure this is needed.
It might interfere with other consumers use of zIndex.
So I think we can remove this, and if a consumer needs to add a `z-index` they can add it using a className which is passed to the component

Reply:
Removed